oci/private/pull.bzl

```diff
@@ -40,19 +40,24 @@ _WWW_AUTH = {
         "service": "registry.docker.io",
     },
     "public.ecr.aws": {
-        "realm": "public.ecr.aws/token",
+        "realm": "{registry}/token",
         "scope": "repository:{repository}:pull",
-        "service": "public.ecr.aws",
+        "service": "{registry}",
     },
     "ghcr.io": {
-        "realm": "ghcr.io/token",
+        "realm": "{registry}/token",
         "scope": "repository:{repository}:pull",
-        "service": "ghcr.io/token",
+        "service": "{registry}/token",
     },
     "cgr.dev": {
-        "realm": "cgr.dev/token",
-        "service": "cgr.dev",
+        "realm": "{registry}/token",
         "scope": "repository:{repository}:pull",
+        "service": "{registry}s",
```

Comment:
the trailing "s" on this line looks suspicious. I wonder if we have e2e tests that fetch from each registry so we know they all work? it would be good to add a simple Azure CR pull in one of the e2e/examples as well.

Reply:
i believe that is a typo.